src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -57,7 +57,25 @@
       return settings.getStatusPlanInfo()
           .then(function (result) {
             vm.extraEmailsSent = 0;
-            vm.resetDate = result.data.endDate;
+            vm.isAccountClosed = result.data.accountClosed;
+            if (result.data.accountEndDate) {
+              vm.resetDate = result.data.accountEndDate;
```

Comment:
In my opinion, resetDate should always refer to current period ending (`result.data.endDate`)